app/android/app/src/main/assets/capacitor.config.json

```diff
@@ -6,10 +6,14 @@
 	"plugins": {
 		"SplashScreen": {
 			"launchShowDuration": 0
+		},
+		"CapacitorHttp": {
+			"enabled": false
 		}
 	},
 	"server": {
-		"allowNavigation": []
+		"allowNavigation": [],
+		"iosScheme": "org.fedarch.faims3"
```

Comment:
I see this value again - how do we manage this for other orgs/deployments?

Reply:
It occurs in code and in config files so we would need to work out how to get the right values in place. Will raise a ticket for this.

Reply:
https://jira.csiro.au/browse/BSS-344